### PR TITLE
IT for debian-adapter with TestDeployment

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #855 
Updated debian adapter and remade integration test for it with `TestDeployment`. 
Extended `MavenITCase.ContainerResultMatcher` with matcher for stdout and ctors.

### CODE REVIEW

**Review by genryxy — DISMISSED** [review]
@olenagerasimova thanks, looks good to me
@g4s8 please check/merge

File: src/test/java/com/artipie/debian/DebianITCase.java [hunk 1]
```diff
@@ -60,106 +46,62 @@
...
-        this.port = this.server.start();
-        final Path setting = this.tmp.resolve("sources.list");
-        Files.write(
-            setting,
-            String.format(
-                "deb [trusted=yes] http://host.testcontainers.internal:%d/my-debian %s main",
-                this.port, DebianITCase.NAME
-            ).getBytes()
+        this.containers.assertExec(
+            "Move debian sources",
```
Comment (g4s8) [inline src/test/java/com/artipie/debian/DebianITCase.java:79]: @olenagerasimova it's failure message that prints if assertion fails, similar to `AssertMatcher.assertThat(msg, ...)`

File: src/test/java/com/artipie/debian/DebianITCase.java [hunk 2]
```diff
@@ -60,106 +46,62 @@
...
-            new Key.From("repos", DebianITCase.NAME), this.storage
-        );
-        new TestResource("debian/pspp_1.2.0-3_amd64.deb")
-            .saveTo(sub, new Key.From("main", "pspp_1.2.0-3_amd64.deb"));
-        new TestResource("debian/Packages.gz").saveTo(
-            sub,
-            new Key.From(String.format("dists/%s/main/binary-amd64/Packages.gz", DebianITCase.NAME))
+    void pushAndInstallWorks() throws Exception {
+        this.containers.assertExec(
+            "Deb package uploaded",
```
Comment (g4s8) [inline src/test/java/com/artipie/debian/DebianITCase.java:88]: @olenagerasimova same here

File: src/test/java/com/artipie/debian/DebianITCase.java [hunk 3]
```diff
@@ -60,106 +46,62 @@
...
+            new MavenITCase.ContainerResultMatcher(),
+            "curl", "http://artipie:8080/my-debian/main/aglfn_1.7-3_amd64.deb",
+            "--upload-file", "/w/aglfn_1.7-3_amd64.deb"
         );
-        this.cntn.execStdout("apt-get", "update");
-        MatcherAssert.assertThat(
-            this.cntn.execStdout("apt-cache", "search", "pspp"),
-            new StringContainsInOrder(new ListOf<>("pspp", "Statistical analysis tool"))
+        this.containers.assertExec(
+            "Apt-get update successful",
```
Comment (g4s8) [inline src/test/java/com/artipie/debian/DebianITCase.java:94]: @olenagerasimova and here

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comments

**Comment by olenagerasimova** [comment]
@g4s8 thanks, corrected messages

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#873` is not assigned, can't get performer

**Pull request merged**